src/lake_modelling/utils/user_inputs.py

```diff
@@ -45,25 +53,30 @@ def get_lim_param(n_months, products):
     col1, col2 = st.columns(2)
 
     lime_prod = col1.selectbox("Choose liming product", (products))
-    lime_dose = col1.number_input('Liming "dose" (mg-lime/l)', min_value=0, value=10)
+    lime_dose = col1.number_input(
+        'Liming "dose" (mg-lime/l)', min_value=0.0, value=10.0
+    )
     lime_month = col1.number_input(
-        "Month in which lime is added, must be less than lenght of simulation",
-        max_value=n_months - 1,
-        value=2,
+        "Month in which lime is added, must be less than length of simulation",
+        max_value=float(n_months) - 1.0,
+        value=1.0,
+        min_value=1.0,
+        step=0.1,
+        format="%.1f",
```

Comment:
Regarding the months, maybe I'm not understanding, why are we changing them to floats here?

Reply:
I don't understand this either. Can you clarify please, @nmarquesin? Thanks!

Reply:
I'm trying to find where I read that we would want the liming month to be a fractional value, not an integer? Maybe it was my mistake. 
@JamesSample, could you confirm that the month liming will be introduced should be an integer value, not fractional (as in, lime is added always at the start of the month, as opposed to any point during the month)?